R/readme.R

```diff
@@ -11,7 +11,13 @@
 #' added to \code{.Rbuildignore}. The resulting README is populated with default
 #' YAML frontmatter and R fenced code blocks (\code{md}) or chunks (\code{Rmd}).
 #'
+#' \code{use_data_list_in_readme_rmd} and \code{use_data_list_in_readme_md}
+#' append a table to the README listing the datasets contained in the package.
 #' @inheritParams use_template
+#' @param stop_if_no_data Logical. If \code{TRUE},
```

Comment:
This seems a bit unnecessary to me. I think it should just silently fail if there's no data, and it should warn if the package isn't installed.